packages/directory/app/styles/navi-directory/components/dir-new-button.less

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+.dir-new-button {
+  &__trigger {
+    background-image: linear-gradient(45deg, #3697F2 0%, #3570F4 100%);
+    border-radius: 2px;
+    color: @navi-white;
+    cursor: pointer;
+    font-family: @font-family-thin;
+    font-size: @font-size-base;
+    margin: 20px 0;
+    padding: 10px;
+    text-align: center;
+    user-select: none;
+    width: 120px;
+  }
+
+  &__content {
+    background: @navi-white;
```

Comment:
I wonder if we need a less mixin for this dropdown too? Im sure we have similar styles for the metric parameter selector too.

Reply:
The location of the dropdown arrow changes the styling pretty drastically. The metric config has the arrow on the bottom of the left side while this dropdown has the arrow on the left of the top side. Can you think of any other components that might use this same style?

Reply:
The before and after is where all the styling for the arrow is. I can add a mix-in without the positioning for the arrow and leave a comment explaining how to add the positioning.